package.json

```diff
@@ -62,10 +62,11 @@
     "eslint-plugin-cypress": "^2.12.1",
     "ethers": "^5.6.9",
     "https-browserify": "^1.0.0",
+    "jest": "^28.1.3",
     "os-browserify": "^0.3.0",
     "process": "^0.11.10",
     "stream-browserify": "^3.0.0",
     "stream-http": "^3.2.0",
     "url": "^0.11.0"
   }
-}
+}
```

Comment:
add empty line

Reply:
<img width="1792" alt="Screen Shot 2022-08-12 at 10 56 56 am" src="https://user-images.githubusercontent.com/88649042/184266054-332abf4d-f922-40b0-b3ae-107a61bf2a14.png">
I tried, but after save it did not accept, also tried it with my teammates.